verilog/analysis/verilog_linter_configuration.cc

```diff
@@ -83,6 +83,46 @@ std::string ProjectPolicy::ListPathGlobs() const {
                        });
 }
 
+bool RuleBundle::ParseConfiguration(absl::string_view text, std::string *error) {
+  // Clear the vector to overwrite any existing value.
+  rules.clear();
+
+  for (absl::string_view part : absl::StrSplit(text, absl::ByAnyChar(",\n"),
```

Comment:
This is the only line that is different when comparing to the previous implementation.